docs/index.rst

```diff
@@ -157,24 +157,27 @@ by a long-term support policy.
 *✓* indicates that the version has been tested and works. *×* indicates that it has not been tested, or
 is known to be incompatible.
 
-===========  === === === === === ===   === === === === === === ====
-django CMS   Python                    Django
------------  -----------------------   ----------------------------
-\            3.9 3.8 3.7 3.6 3.5 3.4   3.2 3.1 3.0 2.2 2.1 2.0 1.11
-===========  === === === === === ===   === === === === === === ====
-4.0.x        ✓   ✓   ✓   ✓   ×   ×     ✓   ✓   ✓   ✓   ×   ×   ×
-3.9.x        ✓   ✓   ✓   ✓   ×   ×     ✓   ✓   ✓   ✓   ×   ×   ×
-3.8.x        ×   ✓   ✓   ✓   ✓   ×     x   ✓   ✓   LTS ×   ×   ×
-3.7.x        ×   ✓   ✓   ✓   ✓   ✓     x   x   ✓   LTS ✓   ✓   LTS
-3.6.x        ×   ×   ✓   ✓   ✓   ✓     x   x   x   ✓   ✓   ✓   ✓
-3.5.x        ×   ×   ✓   ✓   ✓   ✓     x   x   ×   ×   ×   ×   ✓
-3.4.5        ×   ×   ×   ✓   ✓   ✓     x   x   ×   ×   ×   ×   LTS
-===========  === === === === === ===   === === === === === === ====
-
-
-
-.. _Python: https://www.python.org
-.. _Django: https://www.djangoproject.com
+===========  ==== === === === === === ===   === === === === === === === ====
+django CMS   Python                         Django
+-----------  ----------------------------   --------------------------------
+\            3.10 3.9 3.8 3.7 3.6 3.5 3.4   4.0 3.2 3.1 3.0 2.2 2.1 2.0 1.11
+===========  ==== === === === === === ===   === === === === === === === ====
+4.2.x        ✓    ✓   ✓   ×   ×   ×   ×     ✓   ✓   ×   ×   ×   ×   ×   ×
+4.0.x        ✓    ✓   ✓   ✓   ×   ×   ×     ✓   ✓   ×   ×   ×   ×   ×   ×
```

Comment:
My apologies for the misinformation regarding 3.11 having the same conditions as 4.0, I forgot about the newer python 3.10 and django 4.0 changes. The v4 branches aren't yet structured in a way that we can start adding those changes. 

```suggestion
4.0.x      ×    ✓   ✓   ✓   ×   ×   ×     ×   ✓   ✓   ✓   ✓   ×   ×   ×
```